tests/library/test_capture.py

```diff
@@ -1,8 +1,18 @@
-from guidance import capture, models, one_or_more, select, guidance
+from guidance import capture, models, one_or_more, select, guidance, user
+import pytest
 
 from ..utils import get_model
 
 
+@pytest.fixture(scope="module")
+def instruct_model(selected_model, selected_model_name):
+    if selected_model_name in ["transformers_phi3cpu_mini_4k_instruct"]:
```

Comment:
Is this the only model for which the test works? I thought that some of the others supported the role tags? Perhaps move the fixture to `conf.py` and call it something like `model_with_role_tags` ?

Reply:
Ah it appears so! It looks like there's both transformers and non transformers of phi3 mini instruct 4k, as well as a couple other instruct versions I recognized. 

Good idea to move it to the conftest.py file though, I can definitely see situations where we could use more tests specifically for models that utilize roles.